core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpReceiverTest.java

```diff
@@ -1258,6 +1261,144 @@ public void testWriterInsertNewSymbolsIntoTableWithExistingSymbols() throws Exce
         });
     }
 
+    @Test
+    public void testDropTable() throws Exception {
+        Assume.assumeTrue(walEnabled);
+        configOverrideMaxUncommittedRows(2);
+        configOverrideWalSegmentRolloverRowCount(2);
+        FilesFacade filesFacade = new TestFilesFacadeImpl() {
+            @Override
+            public int openRW(LPSZ name, long opts) {
+                if (Chars.endsWith(name, "weather~2" + Files.SEPARATOR + "wal1" + Files.SEPARATOR + "1.lock")) {
+                    mayDrainWalQueue();
+                    dropWeatherTable();
```

Comment:
If I add line 

```java
Os.sleep(1000);
```

data assertion fail. Can you fix or explain why?

![Screenshot 2023-01-20 at 14 33 32](https://user-images.githubusercontent.com/2159629/213724126-4aaf1987-98c3-4d99-b978-60b93af6d49c.png)

Reply:
Fixed in https://github.com/questdb/questdb/pull/2926/commits/a044c01937630b933b4e5704a1a949a865c6c09a